src/components/pages/edit-doc.js

```diff
@@ -54,13 +54,9 @@ class EditDoc extends React.Component {
 
     autoBind(this);
 
-    const { editorFactory, rendererFactory, documentApiClient, initialState } = this.props;
+    const { initialState } = this.props;
     const { documentRevision, proposedSections } = initialState;
 
-    this.editorFactory = editorFactory;
```

Comment:
Uh, more leftovers from menus 😬

Reply:
Now the good linter tells us more about unused code